index.js

```diff
@@ -64,7 +65,17 @@ module.exports = (filename, options) => {
 		}
 
 		(async () => {
-			const data = await getStream.buffer(zip.outputStream);
+			let data;
+			try {
+				data = await getStream.buffer(zip.outputStream, {maxBuffer: BufferConstants.MAX_LENGTH});
```

Comment:
As commented in https://github.com/sindresorhus/gulp-zip/pull/118#discussion_r471084321, this should use `file.isStream()`. It should not automatically switch. That could cause problems.